### PR TITLE
Improve websocket rate limits - Closes #722

### LINKED ISSUE #722: Improve WebSocket rate limits
### Description

Improve socket.io rate limits by preventing one connected user from sending multiple requests in a row.

Follow [this article](https://socket.io/docs/v4/performance-tuning/) to improve overall performance and use available techniques.

### Acceptance Criteria

- socket.io layer is able to prevent users from overusing the service and causing heavy load on the core service.

### PR DESCRIPTION
**Pull request by nagdahimanshu**
### What was the problem?
This PR resolves #722 

### How was it solved?
- [x] Implement websocket rate limits (rate-limiter-fiexible)
- [x] Configure WS rate limit
- [x] Update [config_options.md](https://github.com/LiskHQ/lisk-service/blob/722-improve-websocket-rate-limits/docs/config_options.md) file

### How was it tested?
Local + Jenkins

### CODE REVIEW

File: ecosystem.jenkins.config.js [hunk 1]
```diff
@@ -32,6 +32,7 @@ module.exports = {
 				PORT: '9901',
 				SERVICE_BROKER: 'redis://localhost:6379/0',
 				STRICT_READINESS_CHECK: true,
+				ENABLE_WS_RATELIMIT: false,
```
Comment (sameersubudhi) [inline ecosystem.jenkins.config.js:35]: Please add the following environment variables as well and please update as a comment if `WS_RATELIMIT_DURATION` is in seconds or millisecs. The same config needs to be updated for `gateway` microservice in `ecosystem.core3.config.js` file as well.
```suggestion
				ENABLE_WS_RATELIMIT: false,
				WS_RATELIMIT_POINTS: 5,
				WS_RATELIMIT_DURATION: 1, // in seconds
```

File: services/gateway/config.js [hunk 2]
```diff
@@ -80,6 +80,15 @@ config.api.enableHttpCacheControl = Boolean(String(process.env.ENABLE_HTTP_CACHE
 // Unless STRICT_READINESS_CHECK env. variable is set false, includeCoreReadiness evaluates to true
 config.includeCoreReadiness = Boolean(String(process.env.STRICT_READINESS_CHECK).toLowerCase() !== 'false');
 
+// configuration for websocket rate limit
+config.websocket = {
+	enableRateLimit: Boolean(String(process.env.ENABLE_WS_RATELIMIT).toLowerCase() !== 'false'),
```
Comment (sameersubudhi) [inline services/gateway/config.js:85]: Let us disable `enableRateLimit` until and unless `ENABLE_WS_RATELIMIT` is explicitly set to `true`.

```suggestion
	enableRateLimit: Boolean(String(process.env.ENABLE_WS_RATELIMIT).toLowerCase() === 'true'),
```

**Review by sameersubudhi — CHANGES_REQUESTED** [review]

**Comment by sameersubudhi** [comment]
@nagdahimanshu Please make sure to update the respective docker environment files as well.

**Review by sameersubudhi — APPROVED** [review]
👍

**Review by MichalTuleja — CHANGES_REQUESTED** [review]
@nagdahimanshu Can you please also mention those settings in the documentation?

**Review by MichalTuleja — APPROVED** [review]

**Pull request merged**